2 - RTB/Documentazione interna/Norme di Progetto/Norme di Progetto.typ

```diff
@@ -665,6 +665,72 @@ Le decisioni vengono dunque così classificate:
+Nello specifico l'impatto può essere "lieve", "medio" o "grave" in base alla sua entità mentre la probabilità di occorrenza viene identificata tramite un valore intero da 1 a 5, dove 1 esprime una probabilità molto bassa, mentre 5 esprime una frequenza attesa sostenuta.\
+Il team definisce approcci di trattamento appropriati, compresi piani di mitigazione specifici.
+
+==== Gestire il profilo di rischio  
+- Definire e registrare le soglie e le condizioni di rischio:
+  le soglie di rischio sono stabilite sulla base della probabilità di occorrenza e dell'impatto. 
+  I rischi con un impatto negativo elevato sono trattati in modo più rigoroso rispetto a quelli con un impatto inferiore;
+
+- Stabilire e mantenere un profilo di rischio:
+  il profilo di rischio contiene informazioni sul contesto della gestione dei rischi, il loro stato, le soglie, le probabilità, le azioni richieste in caso di occorrenza e le conseguenze previste.
+  Sarà aggiornato in modo tempestivo in risposta ai cambiamenti nelle condizioni del progetto;
+
+- Fornire il profilo di rischio rilevante agli interessati in base alle loro esigenze:
+  il profilo di rischio verrà all'occorrenza discusso nei meeting interni e/o esterni e tutte le parti interessate allo stato attuale dei rischi e delle azioni di trattamento possono consultare il documento dedicato "Analisi Dei Rischi.pdf" presente in: https://github.com/Error-418-SWE/Documenti-old/blob/develop/1%20-%20Candidatura/Analisi%20dei%20rischi/Analisi%20dei%20rischi.pdf .
+  // riferimenti alla documentazione temporanei 
+
+==== Analizzare i rischi
+Questa attività consiste nei seguenti compiti:
+
++ identificare i rischi nelle categorie descritte nel contesto della gestione dei rischi:\ l'identificazione avviene durante tutte le fasi di sviluppo, inoltre, i rischi emergono dall'analisi delle misurazioni di qualità dei processi e del sistema software in evoluzione;
+
++ stimare la probabilità di occorrenza e le conseguenze di ciascun rischio identificato:\ per ogni rischio identificato, viene stimata la probabilità di occorrenza e le conseguenze, che tipicamente coinvolgono impatti tecnici, di pianificazione, di costo o di qualità;
+
++ valutare ciascun rischio rispetto alle sue soglie di rischio:\ ogni rischio viene valutato rispetto alle soglie di rischio stabilite, determinando se il rischio supera tali soglie e richiede trattamenti specifici.
+
+==== Trattare i rischi
+Questa attività consiste nei seguenti compiti:
+
++ per ciascun rischio che non soddisfi la sua soglia di tolleranza, definire e registrare le strategie e le misure di trattamento consigliate:\ per i rischi che superano le soglie prestabilite, vengono definite strategie di trattamento, che possono includere l'eliminazione del rischio, la riduzione della sua probabilità o gravità, o l'accettazione del rischio. Vengono anche registrate le misure che forniranno informazioni sull'efficacia delle alternative di trattamento.
```

Comment:
Vengono anche registrate le misure che forniranno informazioni sull'efficacia delle alternative di trattamento.

Lo facciamo davvero? Credo siano cose da Piano di Qualifica ma non monitoriamo l'efficacia delle strategie di gestione dei rischi. Io proporrei di eliminare quest'ultima frase, ma qui mi rimetto a @TodescoMattia

Reply:
non teniamo nessun documento ufficiale con quanto riportato, non ci riguarda direi

Reply:
io ho inteso questo punto come un "nella parte dove scriviamo come trattare i rischi in caso si verifichino, specifichiamo cosa implica l'implementazione di tale trattamento" banalmente scrivendo se in quel modo si sistema il rischio o no. Riformulo la frase in "Vengono anche registrate informazioni sull'efficacia delle alternative di trattamento."